src/main/java/ca/warp7/frc2020/Robot.java

```diff
@@ -59,10 +59,10 @@ public void robotInit() {
                 DriveTrain.getInstance(),
                 Flywheel.getInstance(),
                 Feeder.getInstance(),
-                Hopper.getInstance()
+                Hopper.getInstance(),
                 // Intake.getInstance(),
                 // Climber.getInstance(),
-                // ControlPanelSpinner.getInstance()
+                ControlPanelSpinner.getInstance()
```

Comment:
the control panel mechanism isn't on the robot yet. adding this would mean that it tries to also connect to the motor, or it would fail. Either comment out this or comment out the motor in the subsystem